sw/host/opentitanlib/src/test_utils/lc_transition.rs

```diff
@@ -183,6 +197,69 @@ pub fn trigger_lc_transition(
     Ok(())
 }
 
+/// Perform a volatile raw unlock transition through the LC JTAG interface.
+///
+/// Requires the `jtag` to be already connected to the LC TAP. Requires the pre-hashed token be
+/// provided (a pre-requisite of the volatile operation. The device will NOT be reset into the
+/// new lifecycle state as TAP straps are sampled again on a successfull transition. However,
+/// the TAP can be switched from LC to RISCV on a successfull transition.
+pub fn trigger_volatile_raw_unlock(
+    transport: &TransportWrapper,
+    jtag: Rc<dyn Jtag>,
+    target_lc_state: DifLcCtrlState,
+    hashed_token: Option<[u32; 4]>,
+    use_external_clk: bool,
+    post_transition_tap: JtagTap,
+) -> Result<()> {
+    // Wait for the lc_ctrl to become initialized, claim the mutex, and program the target state
+    // and token CSRs.
+    setup_lc_transition(jtag.clone(), target_lc_state, hashed_token)?;
+
+    // Configure external clock and set volatile raw unlock bit.
+    if use_external_clk {
+        jtag.write_lc_ctrl_reg(
+            &LcCtrlReg::TransitionCtrl,
+            LcCtrlTransitionCtrl::EXT_CLOCK_EN.bits()
+                | LcCtrlTransitionCtrl::VOLATILE_RAW_UNLOCK.bits(),
+        )?;
+    } else {
+        jtag.write_lc_ctrl_reg(
+            &LcCtrlReg::TransitionCtrl,
+            LcCtrlTransitionCtrl::VOLATILE_RAW_UNLOCK.bits(),
+        )?;
+    }
```

Comment:
Something like this avoids some repetition here?
```suggestion
    let mut ctrl = LcCtrlTransitionCtrl::VOLATILE_RAW_UNLOCK;
    if use_external_clk {
        ctrl |= LcCtrlTransitionCtrl::EXT_CLOCK_EN;
    }
    jtag.write_lc_ctrl_reg(&LcCtrlReg::TransitionCtrl, ctrl.bits());
```

Reply:
fixed.